src/fr-device/device-profile.mock.data.ts

```diff
@@ -0,0 +1,119 @@
+const expectedJsdom = {
+  identifier: '',
+  metadata: {
+    hardware: {
+      display: {
+        width: 0,
+        height: 0,
+        pixelDepth: 24,
+        angle: '',
+      },
+      cpuClass: null,
+      deviceMemory: null,
+      hardwareConcurrency: 16,
+      maxTouchPoints: null,
+      oscpu: null,
+    },
+    browser: {
+      appName: 'Netscape',
+      // eslint-disable-next-line
```

Comment:
Are these comments added to disable the linter from reporting the object not in alphabetical order?

Reply:
Hmm, good question. I don't remember, to be honest. I'll have to remove them and see what's reported. These are literally just copied from browsers for mocking, so I didn't spend much time on them.